### PR TITLE
static: resolving Aliases

### LINKED ISSUE #11: bug: Handling `Alias`
### Description of the bug
Members of `BaseModel` can be exposed as `Alias`, which are not resolved.

### PR DESCRIPTION
**Pull request by miili**
Resolving an alias to its `Alias.final_target`

closes #11

### CODE REVIEW

**Comment by miili** [comment]
Well, just seeing now that this is a duplicate of https://github.com/mkdocstrings/griffe-pydantic/pull/7

File: src/griffe_pydantic/static.py [hunk 1]
```diff
@@ -87,6 +89,7 @@ def process_attribute(attr: Attribute, cls: Class, *, processed: set[str]) -> No
         kwargs["default"] = attr.value
 
     if attr.name == "model_config":
+        kwargs.pop("ignored_types", None)  # Ignored types cannot be serialized yet.
```
Comment (miili) [inline src/griffe_pydantic/static.py:92]: This resolved to an `ExprTuple` which cannot be serialized by to `str`. Here is the Pydantic signature:

```py
class Model(BaseModel)
    model_config = ConfigDict(ignored_types=(cached_property,))
```

Comment (pawamoy) [reply src/griffe_pydantic/static.py:92]: Can you open a new issue describing the problem, with a reproducible code example?

**Comment by pawamoy** [comment]
Closed in favor of #7 :slightly_smiling_face:

**Pull request closed without merging**